packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [20.0.0]
+
+### Changed
+
+- **BREAKING:** Bump peer dependency `@metamask/keyring-controller` from `^18.0.0` to `^19.0.0` ([#4915](https://github.com/MetaMask/core/pull/4956))
+- **BREAKING:** Bump `@metamask/keyring-api` from `^8.1.3` to `^10.1.0` ([#4948](https://github.com/MetaMask/core/pull/4948))
+  - Bump the peer dependency `@metamask/providers` from `^17.2.0` to `^18.1.0`.
+- Bump `@metamask/eth-snap-keyring` from `^4.3.6` to `^5.0.1` ([#4948](https://github.com/MetaMask/core/pull/4948))
+- Bump `@metamask/snaps-utils` from `^4.3.6` to `^8.3.0` ([#4948](https://github.com/MetaMask/core/pull/4948))
+- Bump `@metamask/snaps-sdk` from `^6.5.0` to `^6.7.0` ([#4948](https://github.com/MetaMask/core/pull/4948))
+- Bump peer dependency `@metamask/snaps-controllers` from `^9.7.0`to `^9.10.0` ([#4948](https://github.com/MetaMask/core/pull/4948))
```

Comment:
Hmm, did we need to do this? We don't usually bump the peer dependency by a minor or patch unless we need to.

Otherwise, this should be marked as a breaking change.

Reply:
This one is on me, I wrongly updated it while updating the `devDependencies` as well. So I think we could revert that line (if we want to avoid marking it as a breaking change. Since this controller has not been released yet, I think that's ok to "downgrade"?

Reply:
Just in case your ok with downgrading this, here's a PR for this:
- https://github.com/MetaMask/core/pull/4959

Reply:
Yeah that makes sense! I will approve that PR.

Reply:
Thanks! I've merged it. I will remove this part from the changelog now.